validphys2/src/validphys/tests/photon/test_structurefunctions.py

```diff
@@ -78,7 +78,7 @@ def test_params():
     test_theory = API.theoryid(theoryid=THEORY_QED)
     theory = test_theory.get_description()
     for channel in ["F2", "FL"]:
```

Comment:
P.S. here `channel` should be renamed to `kind`  to be consistent with the naming in `yadism` (and by extension I assume everywhere else).